FakeRate/app_region/SS.cc

```diff
@@ -0,0 +1,138 @@
+#include "SS.h"
```

Comment:
I don't understand why adding this file is needed. It causes massive changes from ss. to ss:: that are very confusing for reviewing diffs and make merges of other commits difficult. Also if we really need this, we should remove the old lepfilter.h/cc